src/Applications/LDAS_App/sample_config_files/LADAS/exeinp.txt.atmensC90

```diff
@@ -0,0 +1,624 @@
+####################################################################################
```

Comment:
@saraqzhang: If we include a full copy of this file (and the corresponding "centralC180" file) in the repo, it will be incredibly difficult to keep these sample files in synch with future changes in the exeinp contents. 

Perhaps we can strip down the sample files to only the LADAS-relevant and non-obvious parameter settings, without all the comment lines.  I'm thinking of MET_PATH, MET_TAG, LADAS_COUPLING, ADAS_EXPDIR, MET_HINTERP, LANDASSIM_DT, LANDASSIM_T0, FIRST_ENS_ID, ENSEMBLE_FORCING, and JOB_SGMT.   Not sure about the model config parameters, perhaps we need CHOOSEMOSFC.  Basically, anything that isn't obvious (such as resolution, experiment ID, experiment domain, start/end times, etc) or matches the default.

Alternatively, we could remove these files from the branch and just put in the GEOSldas Wiki. 

PS: Where does NUM_LDAS_ENSEMBLE come from?  Is this connected to ADAS ensemble size through the fvsetup script?

Reply:
If we have a strip-down version of  expinp.txt,  it can be put in the the GEOSldas Wiki along with ReadeMe_LADAS. On the other hand,   user cannot use it directly in ldas setup, we need to instruct user to run ldas_setup sample --exeinp  first to generate a complete expinp txt file, then to edit it according  to the items in the strip-down version.

Reply:
For  the case coupled with atmens, NUM_LDAS_ENSEMBLE  needs to be the same as in ADAS Nens.   For the case coupled with central, it is a default (24) in ldas.

Reply:
similar situation for batinp.txt,  we need to instruct user to run ldas_setup  sample --batinp first to get a file to work with.

Reply:
> If we have a strip-down version of expinp.txt, [...] user cannot use it directly in ldas setup, we need to instruct user to run ldas_setup sample --exeinp first to generate a complete expinp txt file, then to edit it according to the items in the strip-down version.

I think the stripped-down version of the exeinp file for LADAS is the better path because it will minimize maintenance going forward.  Unless we are very diligent about keeping up the LADAS-specific exeinp.txt sample files, the user will not be able to use them directly in ldas_setup either.  For the LADAS, the instructions should follow the general guidance of setting up LDAS (incl. the creation of a sample exeinp file through ldas_setup), and minimize the LADAS-specific instructions to only the differences or custom settings that apply to the LADAS.  Please put together a stripped-down version of the exeinp settings for LADAS.